### DISCUSSION TITLE
How to make scratchpad shown on the fullscreen mode.

### DISCUSSION THREAD

**Discussion by Certainty1396** [i3/i3#6286] [Q&A] [answered]
When browsing sites using firefox in fullsreen sometimes some things would be noted down in the notes app in the scratchpad. In my opinion, the scratchpad is used as a notebook according to the guides, but if every time you open the scratchpad window i3 will automatically disable fullscreen, and you have to reenter fullscreen again. It sounds not that convenience.

**Comment by okraits** [comment, marked as answer]
Normally if a window is in fullscreen state, it is the only window visible (using all available space, as the word fullscreen says) and in the foreground, hiding all other windows.
So if another window (the scratchpad window) would be in front of the fullscreen window, that would be rather unusual and contradicting the concept of a fullscreen window.
I guess that is why i3 is reverting the fullscreen state if you show the scratchpad window.

So what you can do is for example re-enable the fullscreen state after firefox gets focus (using ipc).
